lib/beaker/dsl/helpers/puppet_helpers.rb

```diff
@@ -60,6 +60,14 @@ def puppet_group(host)
         # 6. Revert Puppet to the pre-test state
         # 7. Testing artifacts are saved in a folder named for the test
         #
+        # @note Whether Puppet is started or restarted depends on what kind of
+        #   server you're running.  Passenger and puppetserver are restarted before.
+        #   Webrick is started before and stopped after yielding, unless you're using
+        #   service scripts, then it'll behave like passenger & puppetserver.  You
+        #   can make passenger and puppetserver (or webrick using service scripts)
+        #   restart after yielding by setting the :restart_when_done flag of the
```

Comment:
This wording makes it sound like the default behavior is for the server to stop when the default is actually to restart.